src/components/CodeRevokeModal/index.jsx

```diff
@@ -140,13 +150,36 @@ class CodeRevokeModal extends React.Component {
     /* eslint-enable no-underscore-dangle */
   }
 
+  doNotEmailField({ input }) {
```

Comment:
Will this checkbox appear in both "New Email" and "From Template" section?

Reply:
My understanding is that we need to show it in both sections, but I will also confirm it with Nathan in the standup today.